synapse_data_warehouse/synapse/dynamic_tables/V2.23.0__accessrequirement_latest.sql

```diff
@@ -0,0 +1,26 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE DYNAMIC TABLE IF NOT EXISTS ACCESSREQUIREMENT_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+    WITH latest_unique_rows AS (
+        SELECT
+            accessrequirementsnapshots.*,
```

Comment:
Why is the table name included in the SELECT statement?

Reply:
That's a good point - there's no need to explicitly declare the table name since there are no other tables we're working with in this context. [Updated](https://github.com/Sage-Bionetworks/snowflake/pull/84/commits/4cd118e72c57d8d7c3a9a7316b632d659c05a41e)!